File: daq/slc/dqm/src/BevelRange.cc
#include "daq/slc/dqm/BevelRange.h"

using namespace Belle2;

BevelRange::BevelRange() : Range()
{

}

BevelRange::BevelRange(const std::string& name) : Range(name, 1, 0)
{

}

BevelRange::BevelRange(const std::string& name, double a, double b)
  : Range(name, a, b)  // y = a*x + b
{
}

BevelRange::~BevelRange() throw()
{

}

std::string BevelRange::getDataType() const throw()
{
  return "R1B";
}

File: daq/slc/extra/javalib/b2dqm/src/b2dqm/core/BevelRange.java
package b2dqm.core;

public class BevelRange extends Range {

	public BevelRange(String name, double a, double b) {
		super(name, a, b); //y = a*x + b
	}

	public BevelRange(String name, double a) {
		super(name, a, 0);//y = a*x 
	}

	public BevelRange() {
		super("", 0, 0);
	}

	@Override
	public String getDataType() {
		return "R1B";
	}

}
